openedx/core/djangolib/js_utils.py

```diff
@@ -0,0 +1,139 @@
+"""
+Utilities for dealing with Javascript and JSON.
+"""
+import json
+
+from django.utils.html import escapejs
+from mako.filters import decode
+from markupsafe import escape
+
+from xmodule.modulestore import EdxJSONEncoder
+
+
+def _escape_json_for_js(json_dumps_string):
+    """
+    Escape output of JSON dumps that is safe to be embedded in a <SCRIPT> tag.
+
+    This implementation is based on escaping performed in
+    simplejson.JSONEncoderForHTML.
+
+    Arguments:
+        json_dumps_string (string): A JSON string to be escaped.
+
+            This must be the output of json.dumps to ensure:
+            1. The string contains valid JSON, and
+            2. That non-ascii characters are properly escaped
+
+    Returns:
+        (string) Escaped JSON that is safe to be embedded in HTML.
+
+    """
+    json_dumps_string = json_dumps_string.replace("&", "\\u0026")
+    json_dumps_string = json_dumps_string.replace(">", "\\u003e")
+    json_dumps_string = json_dumps_string.replace("<", "\\u003c")
+    return json_dumps_string
+
+
+def escape_jsoon_for_js(obj, cls=EdxJSONEncoder):
```

Comment:
I would suggest any of these as alternatives:

escape_python_for_js
escape_python_object_for_js
escape_for_js